test_SymmetrisedGradientOperator2 errors in SIRF-SuperBuild CI

The [test ](https://github.com/SyneRBI/SIRF-SuperBuild/runs/5142145504?check_suite_focus=true#step:8:831)fails with the following error on Ubuntu 20.04.3 LTS and GCC 9.3.0, numpy [1.18](https://github.com/SyneRBI/SIRF-SuperBuild/runs/5142145504?check_suite_focus=true#step:3:1287).

https://github.com/TomographicImaging/CIL/blob/cf15c7a2218316e22905efd92d155bccaabb6677/Wrappers/Python/test/test_Operator.py#L508-L522

Notice that this is a CIL unit test error. Numpy is installed by pip, so it's different from our normal conda installed.

```
======================================================================
FAIL: test_SymmetrisedGradientOperator2 (test_Operator.TestGradients)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/SIRF-SuperBuild/SIRF-SuperBuild/build/sources/CIL/Wrappers/Python/test/test_Operator.py", line 510, in test_SymmetrisedGradientOperator2
    numpy.testing.assert_almost_equal(lhs2, rhs2, decimal=4)
  File "/home/runner/virtualenv/lib/python3.8/site-packages/numpy/testing/_private/utils.py", line 601, in assert_almost_equal
    raise AssertionError(_build_err_msg())
AssertionError: 
Arrays are not almost equal to 4 decimals
 ACTUAL: -31.290825605392456
 DESIRED: -31.291176676750183

----------------------------------------------------------------------
```

--- a/Wrappers/Python/test/test_Operator.py
+++ b/Wrappers/Python/test/test_Operator.py
@@ -503,7 +503,7 @@ class TestGradients(CCPiTestClass):
         rhs = u1.dot(E1.adjoint(w1))
         print ("lhs {} rhs {}".format (lhs, rhs))
         # self.assertAlmostEqual(lhs, rhs)
-        numpy.testing.assert_almost_equal(lhs, rhs, decimal=4)
+        numpy.testing.assert_allclose(lhs, rhs, rtol=1e-3)
             
     def test_SymmetrisedGradientOperator2(self):        
         ###########################################################################
@@ -519,7 +519,7 @@ class TestGradients(CCPiTestClass):
         lhs2 = E2.direct(u2).dot(w2)
         rhs2 = u2.dot(E2.adjoint(w2))
             
-        numpy.testing.assert_almost_equal(lhs2, rhs2, decimal=4)
+        numpy.testing.assert_allclose(lhs2, rhs2, rtol=1e-3)
         
     def test_SymmetrisedGradientOperator2a(self):        
         ###########################################################################

